src/server/game/Server/Packets/AreaTriggerPackets.h

```diff
@@ -83,6 +83,16 @@ namespace WorldPackets
             Optional<AreaTriggerMovementScriptInfo> AreaTriggerMovementScript;
             ObjectGuid TriggerGUID;
         };
+
+        class AreaTriggerPlaySpellVisual final : public ServerPacket
+        {
+        public:
+            AreaTriggerPlaySpellVisual() : ServerPacket(SMSG_AREA_TRIGGER_PLAY_SPELL_VISUAL, 20) { }
```

Comment:
use `16 + 4` instead of `20`, makes it more clear when reading. Compiler takes care of optimizing it